### DISCUSSION TITLE
分享你们的绘图机器人！

### DISCUSSION THREAD

**Discussion by shigma** [koishijs/novelai-bot#75] [Show and tell]
欢迎大家在这里分享你们使用 NovelAI Bot 搭建的绘图机器人！把服务器链接评论在下方，让更多的人来体验吧！

**Comment by MirrorCY** [comment]
QQ群机器人：2772315229

**Reply by MirrorCY** [reply]
QQ群 767973430

**Reply by MirrorCY** [reply]
bot挂啦 大家爆图太猛了

**Reply by MirrorCY** [reply]
群还在 可以去玩~

**Reply by MirrorCY** [reply]
![image](https://user-images.githubusercontent.com/37006258/215836622-b0d47b1e-3545-4c3b-a573-41123d1815f0.png)

**Reply by yisi1999** [reply]
1

**Reply by MirrorCY** [reply]
2

**Comment by MirrorCY** [comment]
开黑啦频道 https://kook.top/6rGEDC

**Reply by MirrorCY** [reply]
新的邀请链接 只能私聊啦 
[https://kook.top/9MxXht](https://kook.top/9MxXht)

**Reply by MirrorCY** [reply]
私聊也炸了，期待解封中

**Reply by MirrorCY** [reply]
活了，可以去玩，但是 QQ 群人多一点

**Comment by MirrorCY** [comment]
Discord 频道: https://discord.gg/w9uDYbBHnx

**Comment by MirrorCY** [comment]
Telegram 频道：https://t.me/+hhcuca5BUIs5YWM1

**Comment by MirrorCY** [comment]
Matrix 房间：https://matrix.elchapo.cn:65500/NovelAi/#/room/#naitest:matrix.elchapo.cn

测试中，诸多不完善，但是可直连
谢谢你安妮~

**Reply by MirrorCY** [reply]
没有注册的注册后再次点击链接即可加入

**Reply by MirrorCY** [reply]
新年更新，硬盘炸了，由于没有备份所以现在这个没了

**Comment by MirrorCY** [comment]
只有 QQ 群还在哩
